src/mscorlib/src/System/Exception.cs

```diff
@@ -83,8 +83,7 @@ protected Exception(SerializationInfo info, StreamingContext context)
             _stackTraceString = info.GetString("StackTraceString");
             _remoteStackTraceString = info.GetString("RemoteStackTraceString");
             _remoteStackIndex = info.GetInt32("RemoteStackIndex");
-
-            _exceptionMethodString = (String)(info.GetValue("ExceptionMethod", typeof(String)));
```

Comment:
What do we lose without this? i.e. how does lack of this manifest to code that serializes and deserializes an exception?